dask_expr/shuffle.py

```diff
@@ -102,6 +103,9 @@ def _simplify_up(self, parent):
                     parent.operand("columns")
                 ]
 
+        if isinstance(parent, Reduction) and "idx" not in type(parent).__name__:
```

Comment:
I think that this should be changed to have an explicit list of allowable reductions, rather than assume everything unseen is fine.